doc/python/webgl-vs-svg.md

```diff
@@ -198,3 +207,99 @@ fig.show()
 ### Reference
 
 See https://plotly.com/python/reference/scattergl/ for more information and chart attribute options!
+
+## Pandas and NumPy for Improved Performance
+
+Improve the performance of generating Plotly figures that use a large number of data points by using NumPy arrays and other objects that can be converted to NumPy arrays, such as Pandas Series and Index objects.
+
+Plotly.py uses Plotly.js for rendering, which supports typed arrays. In Plotly.py, NumPy array and NumPy-convertible arrays are base64 encoded before being passed to Plotly.js for rendering.
+
+### Arrays and Data Types Supported
+
+The following types of objects in Python are supported for improved performance:
+
+- Numpy `numpy.ndarray` objects.
+- Pandas Index, `pandas.Index`, or Series, `pandas.Series`, objects.
+- Array objects that can be converted to `numpy.ndarray` objects. i.e., they implement `"__array__"` or `"__array_interface__"` and return a `numpy.ndarray`.
+
+The following array data types are supported:
+
+- int8
+- uint8
+- int16
+- uint16
+- int32
+- uint32
+- float32
+- float64
+- int64*
+- uint64*
+
+*If the array dtype is **int64** and **uint64**, often the default dtype for arrays in NumPy when no dtype is specified, those dtypes will be changed to other types internally by Plotly.py where possible.
```

Comment:
We could add: to avoid this conversion it is suggested to declare the type of integer.

Reply:
Good point. Added link.